data-transformers/transformers/statsTransformer.py

```diff
@@ -21,6 +21,7 @@
 trialID = sys.argv[5]
 experimentID = trialID.split("_")[0]
 containerPropertiesID = sys.argv[7]
+containerID = filePath.split("/")[-1].split("_")[0]
```

Comment:
@VincenzoFerme And then the analysers won't have this information, and in the unlikely case of a conflict (which is the whole reason why you want the use of uuid) you won't be able to distinguish the containers, because if two docker containers end up with the same ID, then they will only be distinguishable by their name at that point, so if anything we might as well use container IDs and names worst case scenario.